site/docs/patterns/international-address-input.mdx

```diff
@@ -0,0 +1,147 @@
+---
+title: International Address Input
+layout: DetailPattern
+sidebar:
+  label: International Address Input
+aliases:
+  - /salt/patterns/international-address-input
```

Comment:
Need resources

Reply:
Release without resource link. However, "International Address Input resources" is stretching page horizontally over 100%, need fix